### PR TITLE
[bitnami/nats] Add support for CLI arguments

### PR DESCRIPTION
**Pull request by madflojo**
### Description of the change

Originally opened as https://github.com/bitnami/containers/pull/31496 this pull request adds support for extra arguments being passed to NATS for the NATS container.

This PR is essentially the same but addresses the original comments.

### Benefits

Adds the ability to pass command line arguments such as `-js` to enable jet stream.

### Possible drawbacks

None

### Applicable issues

- fixes #

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by madflojo** [comment]
> Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

Updated the commit

File: bitnami/nats/2/debian-11/rootfs/opt/bitnami/scripts/nats/run.sh [hunk 1]
```diff
@@ -16,8 +16,9 @@ set -o pipefail
 
 declare nats_cmd="nats-server"
 which "$nats_cmd" >/dev/null 2>&1 || nats_cmd="gnatsd"
-declare -a args=("-c" "$NATS_CONF_FILE")
-args+=("$@")
+declare -a args=($NATS_EXTRA_FLAGS)
+
+args+=("-c" "$NATS_CONF_FILE" "$@")
```
[the remark this answers is not in the document]
Comment (rafariossaa) [reply bitnami/nats/2/debian-11/rootfs/opt/bitnami/scripts/nats/run.sh:21]: @madflojo Could you take a look to my suggestion ?

[the remark this answers is not in the document]
Comment (madflojo) [reply bitnami/nats/2/debian-11/rootfs/opt/bitnami/scripts/nats/run.sh:21]: Sure I’m good with the suggestion and I will make the updates

[the remark this answers is not in the document]
Comment (madflojo) [reply bitnami/nats/2/debian-11/rootfs/opt/bitnami/scripts/nats/run.sh]: It's been updated. I did need to re-add the CONF_FILE bits, so please feel free to validate this matches what you were thinking. I did test it, passing two arguments.

**Review by rafariossaa — APPROVED** [review]
Thanks for updating it.
LGTM.

**Pull request merged**